data/Makefile.in

```diff
@@ -365,6 +394,11 @@ R_DEP           := $(R_OBJ:%.o=%.d)
 .PHONY: rec
 rec: $(R_TARGET)
 
+# if the current compiler is not GCC 4 or 5, then enable -no-pie
+ifeq ($(call IS_GCC_4_OR_5,$($(R_CUR_COMPILER))),0)
+  LD_REQUIRED += -no-pie
+endif
+
```

Comment:
This is when the Makefile is in recursive mode, i.e. only compiling one of the full run executables -- so it is safe to add to `LD_REQUIRED` only for `R_CUR_COMPILER`